Cesium.Preprocessor/ConditionExpressions/BinaryExpression.cs

```diff
@@ -48,7 +48,7 @@ public string EvaluateExpression(IMacroContext context)
 
         int Parse(Location location, string macrosValue)
         {
-            if (Regex.IsMatch(macrosValue, $"^(0|[1-9][0-9]*)$"))
+            if (Regex.IsMatch(macrosValue, $"^(-?|\\+?)(0|[1-9][0-9]*)$"))
```

Comment:
_(Minor)_ I believe we can simplify this to `(-|\\+)?`

In plain English, the current code says "either an optional minus sign or an optional plus sign", while the new one would say "a minus or plus sign, optional", which I think better describes the intention.